Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Throwables/boomerang.yml

```
@@ -0,0 +1,44 @@
+  components:
+  - type: LandAtCursor
+  - type: Sprite
+    sprite: _Goobstation/Objects/Weapons/Throwable/boomerang.rsi # GoobStation Edit
+    layers:
+    - state: icon
+  - type: DamageOtherOnHit
+    damage:
+      types:
+        Blunt: 4
+  - type: StaminaDamageOnCollide
+    damage: 10 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)